app/views/overview/_deployment.html

```diff
@@ -1,8 +1,8 @@
 <div class="overview-tile" ng-class="{ 'deployment-in-progress': inProgressDeployment }">
   <ng-include src="'views/overview/_service-header.html'"></ng-include>
   <div class="overview-tile-header">
-    <div class="rc-header">
-      <div>
+    <div class="rc-header" ng-class="{ 'rc-header-shield' : activeReplicationController}">
```

Comment:
@sg00dwin You can test with

https://github.com/openshift/origin-web-console/blob/master/test/testdata/k8s-deployments.yaml